tools/validate_configs/test_configs/htcondor-pool.yaml

```diff
@@ -1,119 +0,0 @@
-# Copyright 2022 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
----
```

Comment:
Would there be any value in keeping this test_config as an alternative htcondor blueprint we can point to?

Reply:
I would rather update the public config with a Packer-based approach when we have that solution automated. It's their in the git history in the meantime.